va/va_dec_av1.h

```diff
@@ -354,7 +354,7 @@ typedef struct  _VADecPictureParameterBufferAV1
      *  Contains a list of indices into ref_frame_map[8].
      *  It specifies the reference frame correspondence.
      *  The indices of the array are defined as [LAST_FRAME – LAST_FRAME,
-     *  LAST2_FRAME – LAST_FRAME, …, ALTREF2_FRAME – LAST_FRAME], where each
+     *  LAST2_FRAME – LAST_FRAME, …, ALTREF_FRAME – LAST_FRAME], where each
      *  symbol is defined as:
      *  enum{INTRA_FRAME = 0, LAST_FRAME, LAST2_FRAME, LAST3_FRAME, GOLDEN_FRAME,
      *  BWDREF_FRAME, ALTREF_FRAME};
```

Comment:
this enum should be {INTRA_FRAME = 0, LAST_FRAME, LAST2_FRAME, LAST3_FRAME, GOLDEN_FRAME,
     *  BWDREF_FRAME,ALTREF2_FRAME, ALTREF_FRAME};